samples/azure-quantum/ising-model/README.md

```diff
@@ -0,0 +1,43 @@
+---
+page_type: sample
+author: cgranade
+description: Simulate evolution under the Ising model with the Trotter–Suzuki decomposition, using the Azure Quantum service
+ms.author: chgranad@microsoft.com
+ms.date: 01/25/2021
+languages:
+- qsharp
+products:
+- qdk
+- azure-quantum
+---
+
+# Simulating the Ising model on hardware with the Trotter–Suzuki decomposition
```

Comment:
Tweak to say "...on quantum hardware..." or replace "hardware" with "on Azure Quantum."
For SEO